llama/llama.go

```diff
@@ -341,21 +363,21 @@ func (b *Batch) IsEmbedding() bool {
 // when the batch was initialized. The other argument will be ignored. Adds to the
 // batch with the given position for the given sequence ids, and optionally instructs
 // to include logits.
-func (b *Batch) Add(token int, embed []float32, pos int, seqIds []int, logits bool) {
+func (b *Batch) Add(token int, embed []float32, pos int, logits bool, seqIds ...int) {
```

Comment:
```suggestion
func (b *Batch) Add(token int, embed []float32, pos int, logits bool, seqIDs ...int) {
```

Reply:
I'll probably do it separately at a later point - both to be consistent rather than just where this touches and also because this patch is already bigger than I would have ideally liked for a late RC.